js/apps/system/_admin/aardvark/APP/frontend/js/views/foxxRepoView.js

```diff
@@ -201,21 +201,23 @@
 
     installFoxxFromStore: function (e) {
       if (window.modalView.modalTestAll()) {
-        var mount, flag;
+        var mount, info, options;
         if (this._upgrade) {
           mount = window.App.replaceAppData.mount;
-          flag = arangoHelper.getFoxxFlag();
         } else {
           mount = window.arangoHelper.escapeHtml($('#new-app-mount').val());
           if (mount.charAt(0) !== '/') {
             mount = '/' + mount;
           }
         }
-        if (flag !== undefined) {
-          this.collection.installFromStore({name: this.toInstall, version: this.version}, mount, this.installCallback.bind(this), flag);
-        } else {
```

Comment:
These two are the same except once without the argument if it's undefined. That's silly because the function doesn't check the argument count and undefined is undefined.